provisioning_templates/provision/kickstart_default.erb

```diff
@@ -80,7 +80,7 @@ skipx
 <% dhcp = !@static -%>
 <% end -%>
 
-network --bootproto <%= dhcp ? 'dhcp' : "static --ip=#{@host.ip} --netmask=#{subnet.mask} --gateway=#{subnet.gateway} --nameserver=#{subnet.dns_servers.join(',')} --mtu=#{subnet.mtu.to_s}" %> --hostname <%= @host %><%= os_major >= 6 ? " --device=#{@host.mac}" : '' -%>
+network --bootproto <%= dhcp ? 'dhcp' : "static --ip=#{@host.ip} --netmask=#{subnet.mask} --gateway=#{subnet.gateway} --nameserver=#{subnet.dns_servers.join(',')} --mtu=#{subnet.mtu.to_s}" %> --hostname <%= @host %><%= os_major >= 6 ? " --device=#{@host.provision_interface.attached_to}" : '' -%> <%= (@host.provision_interface.tag.present? && @host.provision_interface.attached_to.present?) ? "--vlanid=#{@host.provision_interface.tag}" : '' %>
```

Comment:
I see a problem here. If attached_to is not set, previously we set this to "mac" but now it will be blank. You need to do if-statement too: `@host.provision_interface.tag.present? && @host.provision_interface.attached_to.present?`. However this line is getting out of control, I recommend to take similar approach as above - create an array, push into it and then join it with spaces. Take attention when editing, this must not break it's very important template :-)